source/common/json/config_schemas.cc

```diff
@@ -1279,9 +1285,8 @@ const std::string Json::Schema::SDS_SCHEMA(R"EOF(
       "hosts" : {
         "type" : "array",
         "items" : {"$ref" : "#/definitions/host"}
-      },
-      "required" : ["hosts"],
-      "additionalProperties" : false
-    }
+      }
+    },
+    "required" : ["hosts"]
```

Comment:
additionalProperties: false is actually a bug in our tests in the repo. What was here was not doing anything, so I fixed it to require hosts to be present.